### ISSUE TITLE
Problems with AppConfig

### ISSUE THREAD

**Issue by benjaoming** [bittner/django-apptemplates#12]
It seems that if an application is specified via AppConfig, it will not work:

```
INSTALLED_APPS = (
    'wiki.apps.WikiConfig'
)
```

For `django-wiki`, this produces errors like `TemplateDoesNotExist: "wiki:wiki/base.html"`. Other variations like specifying the app config `{% extends "wiki.apps.WikiConfig:wiki/base.html" %}` aren't working neither.

Specified as just the app, it works:

```
INSTALLED_APPS = (
    'wiki'
)
```

I kind of gather from the implementation that it's assumed that the entry of `INSTALLED_APPS` is importable models and not AppConfig class paths?

https://github.com/bittner/django-apptemplates/blob/master/apptemplates/__init__.py#L18

**Comment by benjaoming** [comment]
Just found out that this works fine with [django-app-namespace-template-loader](https://github.com/Fantomas42/django-app-namespace-template-loader), and I'm good switching to that :)

**Comment by bittner** [comment]
Thanks for the hint!

The issue seems to be identical to https://github.com/Fantomas42/django-app-namespace-template-loader/issues/10.

**Comment by benjaoming** [comment]
You can see the fix in the diffset between 0.3 and 0.3.1 where it was fixed: https://github.com/Fantomas42/django-app-namespace-template-loader/compare/0.3...v0.3.1

**Comment by bittner** [comment]
Do you want to try a PR?

**Comment by benjaoming** [comment]
No, I switched to the other app -- remember https://github.com/Fantomas42/django-app-namespace-template-loader/pull/15 ? ;)

**Comment by bittner** [comment]
Just checking. Thanks for reporting, anyway.
